README.md

```diff
@@ -4,27 +4,46 @@ This is an implementation of TPC-H in the [MyriaL language](http://myria.cs.wash
 
 ## Setup
 
-`tpch-myrial` requires TPC-H qgen to generate SQL queries and [Raco](https://github.com/uwescience/raco) to generate the query plans.
+`tpch-myrial` requires TPC-H's `qgen` program to generate SQL queries and [Raco](https://github.com/uwescience/raco) to generate the query plans.
+
+### Install RACO
 
-- [Download qgen](http://www.tpc.org/tpch/tools_download/dbgen-download-request.asp)
 - Install Raco by following its README.md and then make sure to set `RACO_HOME` to the directory of your Raco
 
+### Install qgen
+
+qgen is a tool provided by the TPC.
+
+- [Download the TPC-H Tools](http://www.tpc.org/tpc_documents_current_versions/current_specifications.asp).
+- Extract them somewhere.
+- Edit `makefile.suite` in the tools in the `dbgen` folder and rename it to `makefile`. Follow the instructions in the file.
+- Run `make`.
+- Inside this repo, create a symbolic link to (or copy) `qcgen` and `dists.dss`.
+
 ## Generate the queries
 
 The queries are expressed as TPC-H `qgen` template files. The following command will run `qgen` on all the templates
 using the test parameters from the TPC-H specification.
 
 ```bash
-python gen_validation.py
+python gen_validation.py --tpch <DIRECTORY_OF_TPCH_TOOLS>
 ```
 
+Note that it is normal for query 13, 20, 21, and 22 to fail because these are not included in this repo.
+
 ## See the plan of a query
 
 ```bash
 $RACO_HOME/scripts/myrial [flag] q1.myl
 ```
 
 Run `$RACO_HOME/scripts/myrial -h` for documentation about the different kinds of output.
+For example, in order to view the optimized logical plan immediately, run
+
+```bash
+$RACO_HOME/scripts/myrial -d q1.myl | xdot
```

Comment:
@dhutchis can you add another command as an example

`$RACO_HOME/scripts/myrial -d q1.myl | dot -Tpdf -o q1.pdf`